### PR TITLE
Branding no experimental

### PR DESCRIPTION
**Pull request by TheOneRing**

### CODE REVIEW

**Comment by micbar** [comment]
@HanaGemela Docs are failing

**Comment by TheOneRing** [comment]
Supersedes https://github.com/owncloud/client/pull/7754

File: src/libsync/theme.h [hunk 1]
```diff
@@ -349,14 +349,20 @@ class OWNCLOUDSYNC_EXPORT Theme : public QObject
...
+    virtual bool showVirtualFilesOption() const;
+
+    /**
+     * @brief Whether to show options considered as experimental.
+     *
      * By default, the options are not shown unless experimental options are
      * manually enabled in the configuration file.
      */
-    virtual bool showVirtualFilesOption() const;
+    virtual bool enableExperimentalFreatures() const;
```
Comment (jnweiger) [inline src/libsync/theme.h]: Its a Freature. But its consistently a Freature. You might want to fix the speling there.

Comment (TheOneRing) [reply src/libsync/theme.h]: 🙈

**Comment by jnweiger** [comment]
Drone fail is due to missing fonts. This was a broken state in the 2.6 branch, when 2.6.2 was forked.
Will be automatically fixed when merging back into 2.6

**Pull request merged**